packages/schema/src/json-schema/compiler.ts

```
@@ -0,0 +1,105 @@
+import { IGeneralFieldState } from '../core';
+import { hasCollected, untracked } from '../reactive';
+import { FormPath, isArr, isFn, isPlainObj, isStr, reduce } from '../shared';
+import { hasOwnProperty, isNoNeedCompileObject, patchStateFormSchema, traverse, traverseSchema } from './shared';
```

Biome (1.9.4)

[error] Do not shadow the global "hasOwnProperty" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)